postgres/client.sls

```diff
@@ -25,23 +22,27 @@ postgresql-client-libs:
       - pkgrepo: postgresql-repo
 {%- endif %}
 
+# Alternatives system
 {%- if 'bin_dir' in postgres %}
+  {% if postgres.linux.altpriority|int > 0 %}
+    {% if grains.os_family not in ('Arch', 'Gentoo', 'MacOS', 'Windows',) %}
```

Comment:
You don't need to hard-code the list of distros in the state.
The top-level `if` statement checks if `bin_dir` parameter has been defined. And it is explicitly defined only for RedHat in `osmap.yaml` for now. It does not exist for any other system.

Same applies to the `server` state.

Reply:
Thats correct. The Jinja is unnecessary here. done.